jirahub/github.py

```diff
@@ -25,44 +24,6 @@ def _parse_datetime(value):
     return value.replace(tzinfo=timezone.utc)
 
 
-_ISSUE_METADATA_PREFIX = "\r\n\r\n<!-- JIRAHUB-ISSUE-METADATA-1.0.0\r\n"
-_ISSUE_METADATA_SUFFIX = "\r\n-->"
-_ISSUE_METADATA_RE = re.compile(re.escape(_ISSUE_METADATA_PREFIX) + ".*?" + re.escape(_ISSUE_METADATA_SUFFIX))
-
-_COMMENT_METADATA_PREFIX = "\r\n\r\n<!-- JIRAHUB-COMMENT-METADATA-1.0.0\r\n"
-_COMMENT_METADATA_SUFFIX = "\r\n-->"
-_COMMENT_METADATA_RE = re.compile(re.escape(_COMMENT_METADATA_PREFIX) + ".*?" + re.escape(_COMMENT_METADATA_SUFFIX))
-
-
-def _encode_metadata(metadata, prefix, suffix):
-    metadata_json = json.dumps(metadata)
-    return prefix + metadata_json + suffix
-
-
-def _decode_metadata(metadata_str, prefix, suffix):
-    assert metadata_str.startswith(prefix)
-    assert metadata_str.endswith(suffix)
-
-    metadata_json = metadata_str[len(prefix) : len(metadata_str) - len(suffix)]
-    return json.loads(metadata_json)
-
-
-def _encode_issue_metadata(metadata):
-    return _encode_metadata(metadata, _ISSUE_METADATA_PREFIX, _ISSUE_METADATA_SUFFIX)
-
-
-def _decode_issue_metadata(metadata_str):
-    return _decode_metadata(metadata_str, _ISSUE_METADATA_PREFIX, _ISSUE_METADATA_SUFFIX)
-
-
-def _encode_comment_metadata(metadata):
-    return _encode_metadata(metadata, _COMMENT_METADATA_PREFIX, _COMMENT_METADATA_SUFFIX)
-
-
-def _decode_comment_metadata(metadata_str):
-    return _decode_metadata(metadata_str, _COMMENT_METADATA_PREFIX, _COMMENT_METADATA_SUFFIX)
-
-
 class _IssueTranslator:
```

Comment:
Will do!

There are `_IssueTranslator` classes in both the jirahub.github and jirahub.jira modules.  They are responsible for mapping the github/jira SDK issue objects to and from jirahub.entities.Issue.  Sara and I thought they'd be easier to work with if they shared the same interface, rather than, for example, accessing the "body" attribute from github and the "description" attribute from jira.

I don't particularly like the name, but couldn't think of anything better...  I noticed I used the word "mapping" in the description above, maybe IssueMapper?

Reply:
`IssueMapper` seems better, yeah I like that